### ISSUE TITLE
why we are including z_e(x)  also for decoder input

### ISSUE THREAD

**Issue by shiva1393** [rosinality/vq-vae-2-pytorch#20]
Hi @rosinality, In vqvae.py line no 65, 
quantize = input + (quantize - input).detach()
Next this quantize variable we are using input to the decoder.
but from paper we have to give only embeddings(e_i) there we shouldn't include z_e(x) means input variable .Please Can you clarify  this question.

**Comment by rosinality** [comment]
quantize = input - input.detach() + quantize.detach(), so input is cancelled at the forward time.
